src/applications/vaos/new-appointment/components/TypeOfCarePage/PodiatryAppointmentUnavailableModal.jsx

```diff
@@ -12,12 +12,16 @@ const PodiatryAppointmentUnavailableModal = ({ onClose, showModal }) => {
   return (
     <VaModal
       id="toc-modal"
+      data-testid="toc-modal"
       status="warning"
       visible
       onCloseEvent={onClose}
       modalTitle={title}
       ariaLabel={title}
       role="alertdialog"
+      primaryButtonText="Ok"
+      onPrimaryButtonClick={onClose}
+      uswds
```

Comment:
Replacing the button element to utilize `<VaModal>`'s `primaryButtonText` property and `onPrimaryButtonClick` custom event